test/storage_snapper_configure_snapper_test.rb

```diff
@@ -22,13 +22,7 @@
 
     Yast::Storage.stub(:GetEntryForMountpoint).with("/").once.and_return(data)
 
-    Yast::SCR.stub(:Execute).and_return(1)
-    Yast::SCR.should_receive(:Execute).exactly(2).times.and_return(0)
-
-    Yast::SCR.stub(:Write).and_return(1)
-    Yast::SCR.should_receive(:Write).exactly(2).times.and_return(0)
-
-    Yast::StorageUtils.ConfigureSnapper()
+    expect(Yast::StorageSnapper.ConfigureSnapper?).to eq true
```

Comment:
Testing for the exact execute parameters is a "change detector" and should be avaided, see http://googletesting.blogspot.de/2015/01/testing-on-toilet-change-detector-tests.html. Such tests have negative value and should be removed.